packages/graphql/src/schemaTypes/objectTypes/gql-CurrentProject.ts

```diff
@@ -75,9 +74,23 @@ export const CurrentProject = objectType({
 
     t.connectionField('specs', {
       description: 'Specs for a project conforming to Relay Connection specification',
-      type: Spec,
-      nodes: (source, args, ctx) => {
-        return ctx.project.findSpecs(source.projectRoot, ctx.appData.currentTestingType === 'component' ? 'component' : 'integration')
+      type: 'Spec',
+      nodes: async (source, args, ctx) => {
+        if (!ctx.appData.currentTestingType) {
+          return []
+        }
+
+        const pattern = await ctx.project.specPatternForTestingType(source.projectRoot, ctx.appData.currentTestingType)
```

Comment:
should cache these instead of hitting filesystem each time but outside scope of this PR